acceptance-tests/requirements.txt

```diff
@@ -0,0 +1,9 @@
+papermill==2.1.3
+requests>=2.20.0
+jupyterhub==1.1.0
+nbresuse==0.3.3
+jupyterlab-git==0.20.0
```

Comment:
but why do we need these for running the tests? presumably `nbresuse`, `jupyterhub`, `jupyterlab-git` are all there for jupyterlab sessions and this is just running the CLI is it not?

Reply:
@rokroskar I am not sure. These were just hardcoded in the image for the tests. I did not add these to the image for the tests. Just moved them to a separate file.

Reply:
sure I just thought we could remove them and see if anything breaks :) but lets leave it for now.

Reply:
ok sure if you want to do that here or in another PR. I thought since we added dependabot it's pointless to be bombarded with PRs for upgrades that we don't actually care about

Reply:
I _think_ all it needs is pandas and papermill: https://renkulab.io/gitlab/renku-tutorial/renku-tutorial-flights/-/blob/master/.tutorial/meta/templates/00-FilterFlights.ipynb

Reply:
Ok so I think that the only thing the acceptance tests need is `papermill`. Now the tests should pass. There was only one test that failed after I removed all the packages and it failed because it could not run `papermill`